src/controllers/breachDetail.js

```diff
@@ -31,7 +37,12 @@ async function breachDetailPage (req, res) {
     partial: breachDetail,
     skipPartialModule: true,
     breach: featuredBreach,
-    breachLogos: req.app.locals.breachLogoMap
+    breachLogos: req.app.locals.breachLogoMap,
+    meta: {
+      title: getMessage('breach-detail-meta-title', { company: featuredBreach.Name }),
+      socialTitle: getMessage('breach-detail-meta-social-title', { company: featuredBreach.Name }),
```

Comment:
Q: Should these be `Name` or `Title`?
https://gist.github.com/pdehaan/37381d2f05180e47b18e38d6e4a23ce8

```js
  { "Name": "17Media",
    "Title": "17" },
  { "Name": "2844Breaches",
    "Title": "2,844 Separate Data Breaches" },
  { "Name": "Abandonia",
    "Title": "Abandonia (2015)" },
  { "Name": "Abandonia2022",
    "Title": "Abandonia (2022)" },

  { "Name": "ABFRL",
    "Title": "Aditya Birla Fashion and Retail" },

  { "Name": "BulgarianNationalRevenueAgency",
    "Title": "Bulgarian National Revenue Agency" },

  { "Name": "CouponMomAndArmorGames",
    "Title": "Coupon Mom / Armor Games" },
```

Reply:
Here's a list of all the `.Name`-vs-`.Title` usages. Looks like the majority are using `.Title` for user facing strings (and `.Name` for slugs and back end stuff):

<details>
<summary><kbd>git grep -En "\.(Name|Title)" src</kbd></summary>

```sh
src/client/js/partials/exposureScan.js:117:  newCard.querySelector('.exposure-scan-breach-company-name').textContent = breach.Title
src/controllers/hibp.js:103:      name: breachAlert.Name,
src/controllers/hibp.js:120:      breachAlertName: breachAlert.Name,
src/db/tables/breaches.js:35:          name: breach.Name,
src/db/tables/breaches.js:36:          title: breach.Title,
src/scripts/kube-jobs/recencyToBreachId.js:142:      const breachName = subBreach?.Name
src/scripts/kube-jobs/recencyToBreachId.js:146:      const breachId = allBreaches.find(b => b.Name === breachName)?.Id
src/scripts/migrationCleanup.js:73:        const breachName = subBreach?.Name
src/scripts/migrationCleanup.js:77:        const breachId = allBreaches.find(b => b.Name === breachName)?.Id
src/scripts/recencyToBreachId.js:70:        const breachName = subBreach?.Name
src/scripts/recencyToBreachId.js:74:        const breachId = allBreaches.find(b => b.Name === breachName)?.Id
src/scripts/syncBreaches.js:22:  seen.add(breach.Name + breach.BreachDate)
src/scripts/syncBreaches.js:50:  return breach.Name !== undefined &&
src/scripts/syncBreaches.js:52:    breach.Title !== undefined &&
src/utils/breachLogo.js:21:  const { className, variableName } = getColorForName(breach.Name)
src/utils/breachLogo.js:25:  return `<span role="img" aria-hidden='true' class='${classNames}' style='${backgroundStyle}'>${breach.Name.substring(0, 1)}</span>`
src/utils/breachResolution.js:121:        companyName: b.Name,
src/utils/breaches.js:52:    const foundBreach = foundBreaches.find(foundBreach => foundBreach.Name === annotatingBreach.Name)
src/utils/hibp.js:278:      foundBreaches = allBreaches.filter(breach => breachedAccount.websites.includes(breach.Name))
src/utils/hibp.js:307:  const foundBreach = allBreaches.find(breach => breach.Name.toLowerCase() === breachName)
src/views/partials/allBreaches.js:10:  <a href="/breach-details/${breach.Name}" class="breach-card">
src/views/partials/allBreaches.js:13:      <span>${breach.Title}</span>
src/views/partials/breachDetail.js:10:  if (['Exactis', 'Apollo', 'YouveBeenScraped', 'ElasticsearchSalesLeads', 'Estonia', 'MasterDeeds', 'PDL'].includes(breach.Name)) {
src/views/partials/breachDetail.js:136:        ${data.breach.Title}
src/views/partials/breachDetail.js:153:          breachTitle: data.breach.Name,
src/views/partials/breaches.js:30:        companyName: breach.Title,
src/views/partials/breaches.js:41:          <span class='breach-company'>${logo} ${breach.Title}</span>
```
</details>

Possibly w/ the exception of /src/views/partials/breachDetail.js#L153:

https://github.com/mozilla/blurts-server/blob/0ea1ef5023e4621edc3870e921882daf20a01f2f/src/views/partials/breachDetail.js#L147-L157

![Screenshot 2023-04-25 at 1 42 55 PM](https://user-images.githubusercontent.com/557895/234399197-1e192081-3eac-4749-8b82-beafa01f273d.png)

Reply:
Thanks for catching that! I switched to `.Title` — also for the instance you found in `breachesDetail.js`.